docs/create_allocator_service.md

```diff
@@ -0,0 +1,435 @@
+```
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: fleet-allocator-ingress
+  namespace: default
+  annotations:
+    kubernetes.io/ingress.class: "gce"
+    kubernetes.io/ingress.allow-http: "false"
+spec:
+  tls:
+  - secretName: allocatorsecret
+  backend:
+    serviceName: fleet-allocator-backend
+    servicePort: 8000
+
+```
+
+
+### 7. Deploy the Ingress Resource  
+The allocator service pod needs to exist and the readiness probe should be passing health checks before the ingress is created.
+
+Deploy the Ingress with this command:
+```
+kubectl apply -f allocator-ingress.yaml
+```
+
+The output should look like this:
+```
+ingress "fleet-allocator-ingress" created
+```
+
+
+### 8. Retrieve the Ephemeral Public IP Address
+After deployment, it will take about a minute for the IP address to be present, and about 5 or 6 minutes before it can start returning data.
+
+Run this command to get the IP address:
+```
+kubectl get ingress fleet-allocator-ingress
+```
+
+The output should look something like this:
+```
+NAME                      HOSTS     ADDRESS         PORTS     AGE
+fleet-allocator-ingress   *         130.211.42.92   80, 443   1m
+```
+
+To learn more about the status of the ingress, run:
+```
+kubectl get ingress fleet-allocator-ingress -o yaml
+```
+
+When the output shows the ingress.kubernetes.io/backends as 'HEALTHY' rather than 'UNHEALTHY' or 'UNKOWN' it is ready.
+```
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  annotations:
+    ingress.kubernetes.io/backends: '{"k8s-be-30055--e34f41617775a061":"HEALTHY"}'
+    ingress.kubernetes.io/https-forwarding-rule: k8s-fws-default-fleet-allocator-ingress--e34f41617775a061
+    ingress.kubernetes.io/https-target-proxy: k8s-tps-default-fleet-allocator-ingress--e34f41617775a061
+    ingress.kubernetes.io/ssl-cert: k8s-ssl-e34f98d0d23b2aca-99c16222402fe1c4--e34f41617775a061
+    ingress.kubernetes.io/url-map: k8s-um-default-fleet-allocator-ingress--e34f41617775a061
+    kubectl.kubernetes.io/last-applied-configuration: |
+      {"apiVersion":"extensions/v1beta1","kind":"Ingress","metadata":{"annotations":{"kubernetes.io/ingress.allow-http":"false","kubernetes.io/ingress.class":"gce"},"name":"fleet-allocator-ingress","namespace":"default"},"spec":{"backend":{"serviceName":"fleet-allocator-backend","servicePort":8000},"tls":[{"secretName":"allocatorsecret"}]}}
+    kubernetes.io/ingress.allow-http: "false"
+    kubernetes.io/ingress.class: gce
+  creationTimestamp: 2018-09-04T20:00:11Z
+  generation: 1
+  name: fleet-allocator-ingress
+  namespace: default
+  resourceVersion: "4127"
+  selfLink: /apis/extensions/v1beta1/namespaces/default/ingresses/fleet-allocator-ingress
+  uid: 218e0102-b07d-11e8-b86f-42010a8e0072
+spec:
+  backend:
+    serviceName: fleet-allocator-backend
+    servicePort: 8000
+  tls:
+  - secretName: allocatorsecret
+status:
+  loadBalancer:
+    ingress:
+    - ip: 130.211.42.92
+
+```
+
+
+### 9. Check the /healthz Endpoint.
```

Comment:
I don't think we need to do this. We don't need to explain health checks here, we're just trying to explain how to do FleetAllocations in code.

Reply:
You are right.  I did find it useful since the Ingress takes so long to reach a Healthy state and have received ssl errors despite the Ingress reporting itself as healthy.  It's a step to save the user some potential frustration, but is unnecessary.
I will edit this out.